.env.example

```diff
@@ -1,3 +1,4 @@
+#ALLOW_HTTP=yes
```

Comment:
Personal preference: I'd prefer to have some kind of default value here (`ALLOW_HTTP=no`) instead of having a commented line. Specially if use add `dotenv-safe` eventually.

Reply:
In that case it'll be better to add extra info in `README.md` to explain the expected values for it.

Despite it looks obvious that the opposite to `no` is be `yes`, I prefer avoid leaving place to misunderstandings.

Reply:
done